RELEASE_NOTES.rst

```diff
@@ -79,6 +88,12 @@ Bugfixes:
 
 For bear writers:
 
+-  A new built-in type is available from ``Setting`` for using inside ``run()``
+   signature: ``url``.
+-  Bears support to invoke the program with custom arguments to check for it's
```

Comment:
This is quite generic now. And only for Lint

So, you could say "Lint based Bears have a new argument which can be set to test if a command runs without errors. This can be used to check things like existence of a R module"